xbmc/guilib/GUIPanelContainer.cpp

```diff
@@ -254,6 +254,14 @@ bool CGUIPanelContainer::OnMessage(CGUIMessage& message)
       SetCursor(0);
       // fall through to base class
     }
+    else if (message.GetMessage() == GUI_MSG_SETFOCUS)
+    {
+        if (message.GetParam1()) // subfocus item is specified, so set the offset appropriately
+        {
+            int item = std::min(GetOffset() + (int)message.GetParam1() - 1, (int)m_items.size() - 1);
```

Comment:
`message.GetParam1()` returns an int so there is no need for the cast.

and please use c++ style casts for `m_items.size()` -> `static_cast<int>(m_items.size() - 1)`